everviz/plugins/_crossplot_plugin/crossplot_control_group_plugin.py

```diff
@@ -0,0 +1,161 @@
+from uuid import uuid4
+import dash_html_components as html
+import numpy as np
+from dash.dependencies import Input, Output
+import plotly.graph_objs as go
+from webviz_config import WebvizPluginABC
+
+from everviz.plugins._crossplot_plugin._dash_components._crossplot_callback import (
+    _crossplot_callback,
+)
+from everviz.plugins._crossplot_plugin._dash_components._crossplot_layout import (
+    _get_layout,
+)
+from everviz.plugins._crossplot_plugin._get_indexed_control import (
+    _identify_indexed_controls,
+)
+from everviz.data.load_csv._get_data import get_data
+import dash_core_components as dcc
+
+
+class CrossplotControlGroupPlugin(WebvizPluginABC):
+    def __init__(self, app, data_path, title="Crossplot"):
+        super().__init__()
+        self.title = title
+        self.graph_id = f"graph-{uuid4()}"
```

Comment:
Maybe we don't want to use a lot of the functions in `WebvizPluginABC` such that we can quickly walk away from webviz if needed in the future but there is an internal functionality for generating UUIDs.

So if you want you can replace `self.graph_id = f"graph-{uuid4()}"` with `self.graph_id = self.uuid("graph")`
The same is true for all the other internal elements

Reply:
The main benefit with `self.uuid("someelement")` is probably that it makes plugins to `webviz-config` ready for https://github.com/plotly/dash/issues/460 when/if that becomes a reality (i.e. `webviz-config` can then make sure the IDs given to the different plugin instances in the app are not only unique, but also deterministic across hot reloads (even with multiple instance of the same plugin in the same app).

Calling `self.uuid("someelement")` multiple times within the same plugin instance gives the same result, e.g. it is strictly not necessary to store the output of `self.uuid("someelement")` in `__init__()` (but might of course be a valid choice of code style regardless).